src/main/java/com/greencity/ui/page/EcoNewsPage.java

```diff
@@ -0,0 +1,36 @@
+package com.greencity.ui.page;
+import lombok.Getter;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class EcoNewsPage extends BasePage {
+//    protected ToolbarComponent toolbarComponent;
+//    protected EcoNewsCardContainer ecoNewsCardContainer;
+
+    @Getter
+    @FindBy(xpath = ".//h1[@class='main-header']")
+    private WebElement headerText;
+
+    @Getter
+    @FindBy(xpath = ".//span[@id='create-button-text']")
+    private WebElement createNewsButton;
+
+    public EcoNewsPage(WebDriver driver) {
+        super(driver);
+//        toolbarComponent = new ToolbarComponent(driver);
+//        ecoNewsCardContainer = new EcoNewsCardContainer();
```

Comment:
Commented code should not be included in the main branch, remove it.
This functionality will be added when it is implemented.